interface-definitions/system-login.xml.in

```diff
@@ -36,6 +36,68 @@
                     </properties>
                     <defaultValue>!</defaultValue>
                   </leafNode>
+                  <node name="otp">
+                    <properties>
+                      <help>2FA OTP authentication parameters</help>
+                    </properties>
+                    <children>
+                      <leafNode name="key">
+                        <properties>
+                          <help>Token Key Secret key for the token algorithm (see RFC 4226)</help>
+                          <valueHelp>
+                            <format>txt</format>
+                            <description>OTP key (base32 encoded secret)</description>
+                          </valueHelp>
+                          <constraint>
+                            <regex>[A-Z2-7]{20,10000}</regex>
+                          </constraint>
+                          <constraintErrorMessage>Key must only include base32 characters and be at least 26 characters long</constraintErrorMessage>
+                        </properties>
+                      </leafNode>
+                      <leafNode name="rate-limit">
```

Comment:
I wonder if we really need to make the attempts be configurable on a "per user level" and not on a more global level? I see no use in having different values for user a and user b

Reply:
As an example:
- User A is a normal user, for him we do 2FA
- User B - some kind of automation system (Ansible, handwritten scripts, control system) - standard authentication, without 2FA

Reply:
Well if hotp will be added I still think that configuration options like a rate limiting should be done on a global level and not on a per user level to not make the CLI explode.

Reply:
I think there are advantages to both approaches.
Global settings added
Now it is possible to configure settings globally for all users.
But if necessary, global settings can be overridden for individual users.
For example, if a user uses a separate OTP generator (hardware or whatever) that produces parameters that don't match the basic settings
Fixed?

Reply:
Documentation PR updated

Reply:
Please place the XML definitions that are now used globally and per user in XML include files so there is only one source of truth.

Reply:
Fixed